package/utils/audit/Makefile

```diff
@@ -0,0 +1,184 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
Please use an SPDX license identifier instead.

Reply:
I have not changed that. For now, I have merged everything from the openwrt core and the package feed. To make it more comprehensible, I would have adjusted the things that are not right, e.g. SPDX and so on, only later.